woocommerce-payments.php

```diff
@@ -91,6 +91,36 @@ function wcpay_jetpack_init() {
 			'priority'      => 5,
 		]
 	);
+
+	// When only WooCommerce Payments is active, minimize the data to send back to WPcom for supporting Woo Mobile apps.
+	$jetpack_config->ensure(
+		'sync',
+		array_merge_recursive(
+			\Automattic\Jetpack\Sync\Data_Settings::MUST_SYNC_DATA_SETTINGS,
```

Comment:
This is recommended by Jetpack team @zinigor 
Ref: p1644325083913829/1644230364.417699-slack-CBG1CP4EN